src/icons/Envelopeicon1/Envelopeicon1.tsx

```diff
@@ -14,13 +14,13 @@ const StyledEnvelopeicon1 = styled.svg`
   }
 `;
 
-export const Envelopeicon1 = ({ className }: Props): JSX.Element => {
+export const Envelopeicon1 = ({ className}: Props): JSX.Element => {
```

Comment:
Could you please rename to be "IconEnvelope" instead of "Envelopeicon1" ?  It will be easier to see in the code that its just an "Icon".

Also, please delete the "className", in general, Icon's should never have styles configured inside, they always need to be wrapped by consumer component.

```suggestion
export const IconEnvelope = (): JSX.Element => {
```